### ISSUE TITLE
Moving panes in vim mode

### ISSUE THREAD

**Issue by janko** [zed-industries/zed#12773]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

In Vim, I sometimes like to change panes between being split horizontally and vertically, depending on whether code goes more vertically or horizontally. For this, I use `ctrl-w shift-{j,k}` (for moving a pane from vertical to horizontal split) and `ctrl-w shift-{h,l}` (for moving a pane from horizontal to vertical split).

Currently, Zed is able to swap existing panes with these shortcuts, but not move the pane in a way that changes the split.

### Environment

Zed: v0.138.6 (Zed)
OS: macOS 14.4.1
Memory: 8 GiB
Architecture: aarch64

**Comment by janko** [comment]
Yes, this is still the behavior in the latest Zed version – `ctrl-w shift-{hjkl}` cannot change between horizontal and vertical split in Zed, like Vim can.

**Comment by hvladev** [comment]
Still not resolved.
